terminal.py

```diff
@@ -316,7 +318,12 @@ def terminal(jobs_cmds: List[str] = None):
 
 if __name__ == "__main__":
     if len(sys.argv) > 1:
-        if ".gst" in sys.argv[1]:
+        if "--debug" in sys.argv:
+            os.environ["DEBUG_MODE"] = "true"
+            sys.argv.remove("--debug")
+        else:
+            os.environ["DEBUG_MODE"] = "false"
+        if len(sys.argv) > 1 and ".gst" in sys.argv[1]:
```

Comment:
Can you remove that first condition, it is already done in line 320, we just need the 2nd part

Reply:
You need it twice because if you remove "--debug" you may be left with only one argument.

Reply:
Gotchaaa! Sry, missed that.